linode_api4/linode_client.py

```diff
@@ -250,6 +254,7 @@ def _api_call(
             )
 
         url = "{}{}".format(self.base_url, endpoint)
+        print(url)
```

Comment:
Is this a debug output?

Reply:
Yup, I'll remove it once this PR is out of draft ^